medical_family/views/medical_family_view.xml

```diff
@@ -1,65 +1,63 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should be an empty line between headers and xml code. This should normally be something like:
````xml
<?xml version="1.0" encoding="utf-8"?>
<!-- Copyright 2016-2017 LasLabs Inc.
     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->

<odoo>
    . . .
````
however there's no license holder specified here originally so the xml version tag by itself is fine.